nav2_controller/src/nav2_controller.cpp

```diff
@@ -266,7 +270,6 @@ ControllerServer::on_cleanup(const rclcpp_lifecycle::State & state)
   odom_sub_.reset();
   vel_publisher_.reset();
   speed_limit_sub_.reset();
-  action_server_.reset();
```

Comment:
Readd this reset, its important for lifecycle bring up and shut downs

Reply:
the line265 and line269 are same, i just remove the duplicated one.